src/data/aws_s3_types.py

```diff
@@ -50,6 +51,15 @@ def _has_action(actions: Any, expected: str) -> bool:
     return type(actions) is list and bool(list(filter(lambda action: _is_action(action, expected), actions)))
 
 
+@dataclass
+class BucketCORS:
+    enabled: bool = True  # assume CORS is enabled by default so that the audit report never brings false positives back
```

Comment:
That would be a false negative actually :)

A false positive would be if we say the bucket fails compliance due to no access, but in truth the bucket is correctly configured.